app/javascript/widget/mixins/specs/useAvailabilityComposable.spec.js

```diff
@@ -0,0 +1,104 @@
+import { describe, it, expect, beforeEach, vi } from 'vitest';
```

Comment:
Please keep this spec under `composables` folder and remove all the mixins related to availability?